website/settings/defaults.py

```diff
@@ -408,6 +408,7 @@ class CeleryConfig:
         'scripts.analytics.run_keen_snapshots',
         'scripts.analytics.run_keen_events',
         'scripts.clear_sessions',
+        'osf.management.commands.purge_files',
```

Comment:
This name, and the file itself, should probably be changed to something more like `delete_withdrawn_or_failed_registration_files`